src/components/Work/OpenSeadragon/Viewer.js

```diff
@@ -115,12 +131,36 @@ class OpenSeadragonViewer extends Component {
       ...customControlIds
     });
     this.openSeadragonInstance.addHandler("page", this.handlePageChange);
-    this.openSeadragonInstance.addHandler("bookmark-url-change", function(
-      event
-    ) {});
-    this.openSeadragonInstance.bookmarkUrl();
+    this.openSeadragonInstance.addHandler("pan", this.handlePanZoomUpdate);
+    this.openSeadragonInstance.addHandler("zoom", this.handlePanZoomUpdate);
+    this.openSeadragonInstance.addHandler("open", this.handleOpenViewer);
```

Comment:
What if on the `open` event, we did: 

`this.openSeadragonInstance.addOnceHandler("open", this.handleOpenViewer);`

Do we need the `tiledImage` and `handleFullyLoaded()` code on lines 148 and 152 for any reason?